--- ares_gethostbyaddr.c.orig	2019-04-16 03:25:38.508802300 +0200
+++ ares_gethostbyaddr.c	2019-04-16 03:26:05.971988700 +0200
@@ -38,7 +38,7 @@
 #include "ares_platform.h"
 #include "ares_private.h"
 
-#ifdef WATT32
+#if defined(WATT32) || defined(WINDOWS_PHONE)
 #undef WIN32
 #endif
 
@@ -213,6 +213,9 @@
 
   if (!PATH_HOSTS)
     return ARES_ENOTFOUND;
+#else
+  const char *PATH_HOSTS;
+  return ARES_ENOTFOUND;
 #endif
 
   fp = fopen(PATH_HOSTS, "r");
