--- megaclient.cpp.orig	2019-04-14 23:45:41.150422700 +0200
+++ megaclient.cpp	2019-04-14 23:45:56.090571300 +0200
@@ -11815,7 +11815,7 @@
                     LOG_debug << "LocalNode change detected on syncupload: " << ll->name << " LNsize: " << ll->size << " LNmtime: " << ll->mtime
                               << " NSize: " << rit->second->size << " Nmtime: " << rit->second->mtime << " Nhandle: " << LOG_NODEHANDLE(rit->second->nodehandle);
 
-#ifdef WIN32
+#if defined(WIN32) && !defined(WINDOWS_PHONE)
                     if(ll->size == ll->node->size && !memcmp(ll->crc, ll->node->crc, sizeof(ll->crc)))
                     {
                         LOG_debug << "Modification time changed only";
